Examples/Tests/ohm_solver_EM_modes/README.rst

```diff
@@ -1,11 +1,56 @@
 .. _examples-ohm-solver-em-modes:
 
-Ohm Solver: Electromagnetic modes
-=================================
+Electromagnetic modes
+=====================
```

Comment:
The reason I changed it is because this page is a child of the "Kinetic-fluid Hybrid Models" examples page:
![image](https://github.com/ECP-WarpX/WarpX/assets/40245517/860f3fd1-6a5a-4d2c-859a-2952792c0b4a)
so my thinking was that it is redundant to specify "Ohm solver" again. But I definitely can see an argument that the "Ohm solver" is a type of fluid-hybrid solver and therefore needs the extra specificity.